src/facade/dragonfly_connection.cc

```diff
@@ -923,7 +936,7 @@ void Connection::DispatchFiber(util::FiberSocketBase* peer) {
       recycle(move(msg));
     }
 
-    evc_bp_.notify();
+    queue_backpressure.ec.notify();
```

Comment:
I'm worried about the overhead of accessing the `thread_local` here in the loop (however pipeline_req_pool_ is thread-local too....)